pmm-app/src/pmm-qan/panel/components/Filters/Filters.tsx

```diff
@@ -26,13 +26,13 @@ export const Filters: FC = () => {
     contextActions,
     panelState: { loadingDetails, rawTime },
   } = React.useContext(QueryAnalyticsProvider);
-  const [filters, loading] = useFilters();
+  const [filters, loading, FILTERS_GROUPS] = useFilters();
```

Comment:
Just wondering - what is the purpose of passing a const `FILTERS_GROUPS` via the hook, rather than the way it was done previously just importing it? As I understood it is still const and values do not change in there?
Also, considering that it is coming from the hook, I would not suggest using all-caps naming here as it is quite misleading and makes you think that `FILTERS_GROUPS` is some sort of the constant but it is not.